src/tools/dtrust-tool.c

```diff
@@ -0,0 +1,316 @@
+/*
+ * dtrust-tool.c: tool for D-Trust cards
+ *
+ * Copyright (C) 2024 mario.haustein@hrz.tu-chemnitz.de
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#include "config.h"
+
+#include <stdio.h>
+#include <string.h>
+
+// clang-format off
+#include "libopensc/opensc.h"
+#include "libopensc/cards.h"
+#include "libopensc/errors.h"
+// clang-format on
+
+#include "util.h"
+
+static const char *app_name = "dtrust-tool";
+
+// clang-format off
+static const struct option options[] = {
+	{ "reader", 1, NULL, 'r' },
+	{ "wait", 0, NULL, 'w' },
+	{ "pin-status", 0, NULL, 's' },
+	{ "check-transport-protection", 0, NULL, 'c' },
+	{ "unlock-transport-protection", 0, NULL, 'u' },
+	{ "help", 0, NULL, 'h' },
+	{ "verbose", 0, NULL, 'v' },
+	{ NULL, 0, NULL, 0 }
+};
+
+static const char *option_help[] = {
+	"Uses reader number <arg> [0]",
+	"Wait for card insertion",
+	"Show PIN status",
+	"Check transport protection",
+	"Unlock transport protection",
+	"This message",
+	"Verbose operation, may be used several times",
+};
+// clang-format on
```

Comment:
I would prefer to format this part according to the clang. I think it should be just one more tab indented, isn't it?

Reply:
I fixed it. Becaus all the other programs used one tab, I would like to introduce some oddness here. But if you like two tabs, I am fine with that.

Reply:
Not that I would like it much, but its better to be consistent.

Please, keep the `options` with the clang-format off -- the indentation of the elements is a weirdness that even I can not handle, do not know why it happens nor how to easily fix it without breaking something else in formatting.

Reply:
OK, I reverted the code-style changes.